exporter/exporter.go

```diff
@@ -22,6 +22,8 @@ type ExporterInstance interface {
 	ID() int
 	Name() string
 	Config() *Config
+	Type() string
+	Attrs() map[string]string
```

Comment:
(possible follow-up) Maybe the `Attrs` and `Config()` can be combined somehow in the future. Atm not obvious why there are two similar sounding things.

Reply:
I think maybe a better approach is to change the `solver.ExporterRequest` to keep array of objects that keep type/attrs instead of raw `ExporterInstance`. Like it was previously with the keys that are now removed.